### DISCUSSION TITLE
Is it possible to iterate through YAML variables?

### DISCUSSION THREAD

**Discussion by Spikeophant** [Dynatrace/dynatrace-configuration-as-code#410] [Archive] [answered]
I want to write JSON templates for a large org, some of the divisions of the org would need 1-n k8's namespaces, and other objects, defined in management zones.  Tagging is not always the easiest thing to accomplish due to large variances in the k8's namespaces.  The team that manages DT instruments at the cluster level, and not everyone has admin access to the DT tenants.  What I think would be useful is if I could have a JSON template for k8's namespace name, and in the YAML file have many namespaces and it would iterate through the namespaces in the YAML when MONACO executes and creates/updates the management zone.  Is this currently possible?  If not I can write a custom Python thing to do this in my CI and generate the JSON files for MONACO to use.

**Comment by patrickpichler** [comment, marked as answer]
Hi,
This is sadly not possible, but there is nothing against writing some custom 
python scripts which create the definitions in the YAML + the JSONs you need.
